app/src/main/java/com/jjswigut/matters/ui/EditMatterFragment.kt

```diff
@@ -0,0 +1,23 @@
+package com.jjswigut.matters.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.jjswigut.matters.databinding.FragmentEditMatterBinding
+
+class EditMatterFragment : Fragment() {
+
+    private lateinit var binding: FragmentEditMatterBinding
```

Comment:
in order to prevent memory leaks, you'll want to null out this binding. 

the way the sample docs have binding laid out works pretty well https://developer.android.com/topic/libraries/view-binding#fragments

Reply:
Dang it. I had it like this but said "let's not do something you don't understand why you're doing." But now I know!